test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -54,101 +65,156 @@
      * on the 'Fail' button and the reason for the failure should be
      * specified in the JDialog JTextArea.
      *
-     * @param title           title of the Frame.
-     * @param instructions    specified instruction that user should follow.
-     * @param maxRowLength    number of visible rows of the JTextArea where the
-     *                        instruction is show.
-     * @param maxStringLength number of columns of the instructional JTextArea
-     * @param timeoutMinutes  timeout of the test where time is specified in
-     *                        minutes.
-     * @throws HeadlessException
-     * @throws InterruptedException
-     * @throws InvocationTargetException
+     * @param title                title of the Frame.
+     * @param instructions         specified instruction that user should follow.
+     * @param rows                 number of visible rows of the JTextArea where the
+     *                             instruction is show.
+     * @param columns              Number of columns of the instructional
+     *                             JTextArea
+     * @param testTimeOutInMinutes timeout of the test where time is specified in
+     *                             minutes.
+     * @throws HeadlessException         HeadlessException
+     * @throws InterruptedException      exception thrown for invokeAndWait
+     * @throws InvocationTargetException exception thrown for invokeAndWait
```

Comment:
I expected a more reasonable descriptions when any of the exceptions may be thrown. Anyway, it's unimportant. :)